go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/NordSecurity/gopenvpn
+
+go 1.18
```

Comment:
No external dependencies at all?

Reply:
Nope, everything is built on top of standard library.